server/controllers/update.js

```diff
@@ -1,25 +1,26 @@
 const Models = require('../models');
+
 module.exports = {
-  create: (req, res) => {
-    Models.Update.createInstance(req.body)
-      .then(() => res.sendStatus(201))
-      .catch(e => res.status(500).json(e));
-  },
+  // create: (req, res) => {
+  //   Models.Update.createInstance(req.body)
+  //     .then(() => res.sendStatus(201))
+  //     .catch(e => res.status(500).json(e));
+  // },
   read: (req, res) => {
     // something happens here
     Models.Update.getAll(req.params.projectId)
       .then(updates => res.json(updates))
       .catch(e => res.status(500).json(e));
   },
-  update: (req, res) => {
-    // something happens here
-    Models.Update.updateInstance(req.body)
-      .then(() => res.sendStatus(200))
-      .catch(e => res.status(500).json(e));
-  },
-  delete: (req, res) => {
-    Models.Update.delete(req.body)
-      .then(() => res.sendStatus(200))
-      .catch(e => res.status(500).json(e));
-  }
+  // update: (req, res) => {
```

Comment:
Will the commented out code be used again ? Or could you delete it altogether ?

Reply:
@vyoung415 Yeah, those are the CRUD operations that still need to be reimplemented in the backend but aren't vital for stress testing.